python/ray/tune/tests/test_tuner_restore.py

```diff
@@ -352,6 +354,62 @@ def test_tuner_restore_from_cloud(ray_start_2_cpus, tmpdir):
     tuner3.fit()
 
 
+@pytest.mark.parametrize(
+    "upload_uri",
+    [None, "memory:///test/test_tuner_restore_latest_available_checkpoint"],
+)
+def test_tuner_restore_latest_available_checkpoint(
```

Comment:
you may need to move this test before the resource exhaustion one... See comment there on that test.